Code/src/main/java/nl/utwente/group10/ui/components/blocks/Block.java

```diff
@@ -93,8 +97,46 @@ public final CustomUIPane getPane() {
      * change.
      *
      * This method should only be called after the Block's constructor is done.
```

Comment:
Dit is toch voor alle methoden zo? Lijkt me vrij pittig om een methode aan te roepen van een object dat nog niet bestaat...